docs/specs/markdown.yml

```diff
@@ -851,3 +851,22 @@ outputs:
       "conceptual": "<p>Check <span class=\"no-loc\"  dir=\"ltr\" lang=\"en-us\">Find</span> for more details.\nThis method can be ran on Windows, and you can also used it in Azure SDK.</p>\n",
       "no-loc": ["method", "SDK"]
     }
+---
+# Keep raw date string
+inputs:
+  docfx.yml: |
+    outputType: pageJson
+  docs/a.md: |
+    ---
+    loc_version: "2021-09-16T12:07:50.2667174Z"
+    other_date_string: "9/16/2012 12:07:50 PM"
+    ---
+    (2021-09-16T12:07:50.2667174Z)Date string should not be parsed by Json.Net.
+outputs:
+  docs/a.raw.page.json: |
```

Comment:
Is checking `a.mta.json` sufficient? Remove checking `.raw.page.json` for brevity.

Reply:
No, the test is mainly to test the `pageMetadata`(a html string that built by docfx) in `.raw.page.json`. When build the html string, we will directly call `JValue.ToString()` for `Date` without the `DateParseHandling.None`, we need make sure the date is not get parsed with this test.

We can remove check for `a.mta.json` and `rawMetadata` check in `.raw.page.json` for brevity, because we are calling serializer with `DateParseHandling.None` to write them, they are currently safe.

@yufeih Let me know if we want to remove them(`a.mta.json`, `rawMetadata` in `.raw.page.json` )

Reply:
Yes, remove `a.raw.page.json/rawMetadata` and `a.mta.json/conceptual`, format `pageMetadata` as multilines like  https://github.com/dotnet/docfx/blob/v3/docs/specs/legacy.yml#L65